handler/handler_test.go

```diff
@@ -113,14 +113,51 @@ func TestHandle(t *testing.T) {
 				So(zebedeeMock.GetPublishedDataCalls()[0].UriString, ShouldEqual, testZebedeeEvent.URI)
 			})
 
-			Convey("Then the expected event search data import event is producer", func() {
+			Convey("Then the expected event search data import event is produced", func() {
```

Comment:
I think that this would also read more nicely if the first 'event' was deleted
```suggestion
			Convey("Then the expected search data import event is produced", func() {
```